Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -1016,13 +1017,220 @@ def set_user_command(args):
+            ipv4_contents = list()
+            ipv6_contents = list()
+            for response in ipv4_responses:
+                content = {
+                    'ID': int(response.get('id', '')),
+                    'Name': response.get('name', ''),
+                    'Type': response.get('type', ''),
+                    'Description': response.get('description', ''),
+                    'Addresses': response.get('addresses', []),
+                    'IpCategories': response.get('ipCategories', []),
+                    'Countries': response.get('countries', [])
+                }
+                ipv4_contents.append(content)
+            for response in ipv6_responses:
+                content = {
+                    'ID': int(response.get('id', '')),
+                    'Name': response.get('name', ''),
+                    'Type': response.get('type', ''),
+                    'Description': response.get('description', ''),
+                    'Addresses': response.get('addresses', []),
+                    'IpCategories': response.get('ipCategories', []),
+                    'Countries': response.get('countries', [])
+                }
+                ipv6_contents.append(content)
+            s_content = {
+                "IPv4": ipv4_contents,
+                "IPv6": ipv6_contents
+            }
+            hr = tableToMarkdown(f"IPv4 Destination groups ({len(ipv4_contents)})", ipv4_contents)
+            hr += tableToMarkdown(f"IPv6 Destination groups ({len(ipv6_contents)})", ipv6_contents)
+            entry = {
+                'Type': entryTypes['note'],
+                'Contents': s_content,
+                'ContentsFormat': formats['json'],
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable': hr,
+                'EntryContext': {'Zscaler.IPDestinationGroup': s_content},
+            }
+        else:
+            try:
+                cmd_url = "/ipDestinationGroups"
+                if exclude_type != '':
+                    cmd_url += f"?excludeType={exclude_type}"
+                responses = http_request('GET', cmd_url).json()
+            except json.decoder.JSONDecodeError as exp:
+                return_error(f'Failed to execute zscaler-list-ip-destination-group command. Error: {str(exp)}')
+            contents = list()
+            for response in responses:
+                content = {
+                    'ID': int(response.get('id', '')),
+                    'Name': response.get('name', ''),
+                    'Type': response.get('type', ''),
+                    'Description': response.get('description', ''),
+                    'Addresses': response.get('addresses', []),
+                    'IpCategories': response.get('ipCategories', []),
+                    'Countries': response.get('countries', [])
+                }
+                contents.append(content)
```

Comment:
You can also use the same function `get_contents` here from the previous comment. 😸

Reply:
done.